src/components/LogIn.js

```diff
@@ -48,6 +48,10 @@ export default class Form extends Component {
       return (<Redirect to={'/home'} />)
     }
 
+    if(localStorage.getItem('authorizedUser')) {
```

Comment:
It's better to declare new variable isUserLoggedIn = localStorage.getItem('authorizedUser';

Reply:
done